homeassistant/components/frontend/storage.py

```diff
@@ -23,6 +23,22 @@ async def async_setup_frontend_storage(hass: HomeAssistant) -> None:
     websocket_api.async_register_command(hass, websocket_get_user_data)
 
 
+async def async_user_store(hass: HomeAssistant, user_id: str) -> tuple[Store, Any]:
```

Comment:
By the looks of it, `store[id]` is a stringly-keyed dict, so I guess this could be made more accurate..?
```suggestion
async def async_user_store(hass: HomeAssistant, user_id: str) -> tuple[Store, dict[str, Any]]:
```

Reply:
Fixed